src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --main-color: #000;
+  --hover-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: var(--main-color);
+}
+
+img {
+  display: block;
+  width: 100%;
+  object-fit: cover;
+  height: auto;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav {
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  position: relative;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: block;
+  height: 100%;
+  color: var(--main-color);
+  text-decoration: none;
+  line-height: 60px;
+}
+
+.nav__link:hover {
+  color: var(--hover-color);
+}
+
+.nav__link.is-active {
+  color: var(--hover-color);
+  cursor: default;
```

Comment:
When we hover on active element cursor also should be pointer

Reply:
I want to clarify for the future, we can’t click on the active element anyway, should the cursor change to a pointer?